packages/bsky/src/feed-gen/with-friends.ts

```diff
@@ -11,26 +11,30 @@ const handler: AlgoHandler = async (
 ): Promise<AlgoResponse> => {
   const { cursor, limit = 50 } = params
   const feedService = ctx.services.feed(ctx.db)
+  const graphService = ctx.services.graph(ctx.db)
 
   const { ref } = ctx.db.db.dynamic
 
-  // @NOTE use of getFeedDateThreshold() not currently beneficial to this feed
-  const keyset = new FeedKeyset(ref('feed_item.sortAt'), ref('feed_item.cid'))
+  const keyset = new FeedKeyset(ref('post.indexedAt'), ref('post.cid'))
+  const sortFrom = keyset.unpack(cursor)?.primary
 
-  // @TODO apply blocks and mutes
   let postsQb = feedService
-    .selectFeedItemQb()
-    .innerJoin('post_agg', 'post_agg.uri', 'feed_item.uri')
-    .where('feed_item.type', '=', 'post')
+    .selectPostQb()
+    .innerJoin('post_agg', 'post_agg.uri', 'post.uri')
     .where('post_agg.likeCount', '>=', 5)
     .whereExists((qb) =>
       qb
         .selectFrom('follow')
         .where('follow.creator', '=', requester)
-        .whereRef('follow.subjectDid', '=', 'originatorDid'),
+        .whereRef('follow.subjectDid', '=', 'post.creator'),
     )
+    .where((qb) =>
+      graphService.whereNotMuted(qb, requester, [ref('post.creator')]),
+    )
+    .whereNotExists(graphService.blockQb(requester, [ref('post.creator')]))
+    .where('post.indexedAt', '>', getFeedDateThreshold(sortFrom))
 
-  postsQb = paginate(postsQb, { limit, cursor, keyset })
+  postsQb = paginate(postsQb, { limit, cursor, keyset, tryIndex: true })
```

Comment:
Can we make `tryIndex` the default now?

Reply:
Hmm, that would be nice!  It's still not used in most places.  Maybe we flip a bunch more on soon, and then after that make it the default?